### PR TITLE
Fix bug where already existing nodes could not be interlinked

### LINKED ISSUE #11: SelfbuildingGraph cannot handle interlinking in already existing nodes
As of now, if the "missing triplet" for two URIs is already in the current graph (because it was the missing triplet for two other URIs), the process will be ignored and no new fact node will be created.

Simplified example on how it could be:
image

### PR DESCRIPTION
**Pull request by SiC42**
This should fix #11 and probably other bugs, as it now allows two nodes with the same content to be recognized as the same node.

### CODE REVIEW

**Comment by SiC42** [comment]
I noticed an issue where making **two nodes with the same content equal** cannot produce answers for questions like "elton john spouse spouse", i.e. questions with the same word twice.
Working as intended? Your call @RicardoUsbeck

**Comment by RicardoUsbeck** [comment]
I think it is fair if we handle that in preprocessing, i.e. "elton john spouse spouse" => "elton john spouse"

**Comment by SiC42** [comment]
Let me rephrase that question:
spouse of elton johns spouse

**Comment by RicardoUsbeck** [comment]
nah, that would be evil. I think we can omit that for now. Can you please write an @ignore unit test for that?
